tests/test_optimizer.py

```diff
@@ -1127,3 +1128,198 @@ def test_optimize_with_single_constraint():
 
     # test depends on verbose > 0
     optimizer.optimize(eq, objective=objectective, constraints=constraints, verbose=2)
+
+
+@pytest.mark.unit
+def test_proximal_jacobian():
+    """Test that JVPs and manual concatenation give the same result as full jac."""
```

Comment:
Can you add some info about speed and memory in the docstring or somewhere else?